packages/cli/src/commands/query.ts

```diff
@@ -226,6 +226,10 @@ export default {
             print.error("-----------------------------------");
           }
         }
+
+        if (errors) {
+          process.exitCode = 1;
+        }
       }
```

Comment:
Why another condition? just add this after printing errors

Reply:
Ye, did some experimenting with types. Fixed now